target/linux/ramips/dts/rt3050.dtsi

```diff
@@ -296,7 +296,7 @@
 	};
 
 	usbphy: usbphy {
-		compatible = "ralink,rt3050-usbphy";
+		compatible = "ralink,rt3352-usbphy";
 		#phy-cells = <0>;
```

Comment:
Please also fix the reset:
```
		resets = <&sysc 22>, <&sysc 25>;
		reset-names = "host", "device";
```
The `device` reset is essential though RT3050 doesn't use it.
https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/tree/drivers/phy/ralink/phy-ralink-usb.c?h=next-20241213#n204

Reply:
Done.